lib/stoplight/error.rb

```diff
@@ -2,10 +2,6 @@
 
 module Stoplight
   module Error
-    HANDLER = lambda do |error|
-      raise error if AVOID_RESCUING.any? { |klass| error.is_a?(klass) }
-    end
-
     AVOID_RESCUING = [
```

Comment:
`NIT` Do we still use `AVOID_RESCUING` after removing the `HANDLER`?

Reply:
Good catch